clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -353,13 +361,21 @@ object GarbageCollector {
     val removed =
       remove(storageNamespace, gcAddressesLocation, expiredAddresses, runID, region, hcValues)
 
-    // BUG(ariels): Write to some other location!
+    val commitsDF = getCommitsDF(runID, gcCommitsLocation, spark)
+    val reportLogsDst = s"${storageNamespace}/_lakefs/logs/gc/summary/"
+    val reportExpiredDst = s"${storageNamespace}/_lakefs/logs/gc/expired_addresses/"
+
+    val time = DateTimeFormatter.ISO_INSTANT.format(java.time.Clock.systemUTC.instant())
+    writeParquetReport(commitsDF, reportLogsDst, time, "commits.parquet")
+    writeParquetReport(expiredAddresses, reportExpiredDst, time)
+    writeJsonSummary(reportLogsDst, removed.count(), gcRules, hcValues, time)
+
     removed
       .withColumn("run_id", lit(runID))
       .write
       .partitionBy("run_id")
       .mode(SaveMode.Overwrite)
-      .parquet(gcAddressesLocation + ".deleted")
+      .parquet(s"${storageNamespace}/_lakefs/logs/gc/deleted_objects/${time}/deleted.parquet")
```

Comment:
Can you reuse the writeParquetReport to writing the deleted objects report? you can adjust it to receive optional partitionedBy param etc

Reply:
I thought about that, but didn't find how I can do just `df.write.parquet(dstPath)` in that case..